matrix/common/request/request_tracker.py

```diff
@@ -42,8 +37,8 @@ def __init__(self, request_hash: str):
     @property
     def is_initialized(self) -> bool:
         try:
-            self.dynamo_handler.get_table_item(DynamoTable.OUTPUT_TABLE,
-                                               request_hash=self.request_hash)
+            self.dynamo_handler.get_table_item(DynamoTable.STATE_TABLE,
+                                               request_id=self.request_id)
```

Comment:
Was this arbitrary? Just wondering

Reply:
Ah, request ID is in this table now

Reply:
Yup request ID is key of both state and cache table